File: C#/RefactoringWorkshop/RefactoringWorkshop/Dispensable/DataClasses/Customer.cs
namespace RefactoringWorkshop.Dispensable.DataClasses
{
    public class Customer
    {
        public string Title { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public Address Address { get; set; }

        public Customer(string title, string firstName, string lastName, Address address)
        {
            Title = title;
            FirstName = firstName;
            LastName = lastName;
            Address = address;
        }
    }
}

File: Java/RefactoringWorkshop/src/main/java/dispensable/dataclasses/Customer.java
package dispensable.dataclasses;

public class Customer {
    private String title;
    private String firstName;
    private String lastName;
    private Address address;

    public Customer(String title, String firstName, String lastName, Address address) {
        this.title = title;
        this.firstName = firstName;
        this.lastName = lastName;
        this.address = address;
    }

    public Address getAddress() {
        return this.address;
    }

    public String getTitle() {
        return title;
    }

    public String getFirstName() {
        return firstName;
    }

    public String getLastName() {
        return lastName;
    }
}
